doc/devel/documenting_mpl.rst

```diff
@@ -988,10 +988,10 @@ Example:
 
 .. TODO: Add section about uploading docs
 
-.. _ReST: http://docutils.sourceforge.net/rst.html
-.. _Sphinx: http://www.sphinx-doc.org
+.. _ReST: https://docutils.sourceforge.io/rst.html
+.. _Sphinx: https://www.sphinx-doc.org/en/master/
```

Comment:
```suggestion
.. _Sphinx: https://www.sphinx-doc.org
```
I think we want to link the top-level site and let it do the redirect to whatever it wants.